app/containers/Activity/ActivityActions.js

```diff
@@ -0,0 +1,20 @@
+import { connect } from 'react-redux'
+import { changeFilter, fetchActivityHistory, updateSearchText } from 'reducers/activity'
+import ActivityActions from 'components/Activity/ActivityActions'
+
+const mapDispatchToProps = {
+  changeFilter,
+  fetchActivityHistory,
+  updateActivitySearchQuery: updateSearchText
+}
+
+const mapStateToProps = state => ({
+  filter: state.activity.filter,
+  filters: state.activity.filters,
```

Comment:
In most cases it's better to avoid variable names that differ in only one letter like `thing` and `thingS` because it's very error prone and harder to read

Reply:
I think we touched on this before. I agree in principal, but the current code style of the app is to do it this way.

I'd like to keep things consistent across the codebase, so if we are going to change it (and I think we should), that should change should be made across the entire codebase as a standalone PR.